faqs/README.md

```diff
@@ -135,18 +135,22 @@ The following prices are exclusive of any local taxes and shipping/handling fees
 | Raspberry Pi 4 Model B 2GB | $35 |
 | Raspberry Pi 4 Model B 4GB | $55 |
 | Raspberry Pi 4 Model B 8GB | $75 |
+| [Raspberry Pi 4 Model B official kits 2/4/8GB](https://www.raspberrypi.org/products/raspberry-pi-4-desktop-kit)  | $100/$120/$140 |
 | Raspberry Pi Zero | $5 |
```

Comment:
Is it worth making _every_ item in this list into a link to the corresponding product-page?
If the "Raspberry Pi 4 Model B official kits" is being collapsed into a single "2/4/8GB" entry, I wonder if it might make sense to do the same for the "Raspberry Pi 4 Model B"? :shrug:  (or perhaps that would make the information too dense)

Reply:
That was one of the questions I was pondering.

Reply:
I'd add links. There's loads of useful product info on these pages as well as links to trustworthy Approved Resellers. Would also collapse the Model B entries I think.

Reply:
Links added.